src/viewport/index.css

```diff
@@ -18,4 +18,15 @@
     padding: 0;
     background-color: white;
     border: solid 1px #979797;
-}
\ No newline at end of file
+}
+
+.tp-dfwv {
+    z-index: 1000;
+}
+
+.tp-rotv {
```

Comment:
Would be awesome to have some comments in here noting (1) that these are for the control pane (2) the difference between these 2 classes (why is z-index specified twice)

Reply:
yes good idea. i'll add something